lib/signal-list-view.coffee

```diff
@@ -10,38 +10,25 @@ class SignalListView extends SelectListView
         super
 
         @basicCommands = [
-            {
-                name: 'Interrupt'
-                value: 'interrupt-kernel'
-                language: null
-            },
-            {
-                name: 'Restart'
-                value: 'restart-kernel'
-                language: null
-            },
-            {
-                name: 'Shut Down'
-                value: 'shutdown-kernel'
-                language: null
-            },
+            name: 'Interrupt'
+            value: 'interrupt-kernel'
+        ,
+            name: 'Restart'
+            value: 'restart-kernel'
+        ,
+            name: 'Shut Down'
+            value: 'shutdown-kernel'
         ]
 
         @wsKernelCommands = [
-            {
-                name: 'Rename'
-                value: 'rename-kernel'
-                language: null
-            },
-            {
-                name: 'Disconnect from'
-                value: 'disconnect-kernel'
-                language: null
-            }
+            name: 'Rename session for'
```

Comment:
Addresses [this comment](https://github.com/nteract/hydrogen/pull/425#discussion_r77356825)